test/sql/function/list/list_contains.test

```diff
@@ -0,0 +1,358 @@
+# name: test/sql/function/list/list_contains.test
+# description: Test list_ccontains function
+# group: [list]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+create table TEST2 (i int[], j int);
+
+statement ok
+insert into TEST2 values ([2,1,3], 2), ([2,3,4], 5), ([1], NULL);
+
+query T
+select list_contains(i, j) from TEST2
+----
+1
+0
+NULL
+
+statement ok
+create table TEST (i int[]);
+
+statement ok
+insert into TEST values ([2,1,3]), ([2,3,4]), ([1]);
+
+query TT
+SELECT i, list_contains(i,1) from TEST;
+----
+[2, 1, 3]	1
+[2, 3, 4]	0
+[1]	1
+
+query TT
+SELECT i, list_contains(i,4.0) from TEST;
+----
+[2, 1, 3]	0
+[2, 3, 4]	1
+[1]	0
+
+statement ok
+DROP table TEST;
+
+statement ok
+create table STR_TEST (i string[]);
+
+statement ok
+insert into STR_TEST values (['a','b','c']), (['d','a','e']), (['b']);
```

Comment:
Just to be sure, but can you also create a test that has strings longer than 12 characters? DuckDB handles these differently.